examples/user_interaction/client/src/App.js

```diff
@@ -23,47 +23,49 @@ class App extends React.Component {
     this.state = { pi: { time: 0, value: "unknown" }, prime_numbers: { time: 0, value: [] } };
     this.handleClick = this.handleClick.bind(this);
     this.runSecondTask = this.runSecondTask.bind(this);
-    this.http = require('http');
+    this.host = 'http://localhost:8088';
   }
 
   handleClick() {
-    const host = 'localhost';
-    const port = 8088;
-    this.http.get({
-      host,
-      port,
-      path: '/sleep'
-    }, (response) => {
-      let data = [];
-      response.on('data', chunk => data.push(chunk));
-      response.on('end', this.runSecondTask);
-    });
+    console.log("Entering handle click.");
+
+    this.runFirstTask();
+  }
+
+  runFirstTask() {
+    const xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = () => {
+      if (xhr.readyState === XMLHttpRequest.DONE) {
+        this.runSecondTask();
+      }
+    };
+    xhr.open('GET', this.host + "/sleep");
+    xhr.send();
   }
 
   runSecondTask() {
```

Comment:
Similarly, what would you think about calling this `callPrimeNumbersApi` or similar?